#! /bin/sh -e
## doc-package.dpatch by Timshel Knoll <timshel@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Some small debian parted-doc fixup.
## DP: Will stay as debian specific change.

. `dirname $0`/DPATCH

@DPATCH@
diff -urNda parted-1.6.12.orig/doc/parted.8 parted-1.6.12/doc/parted.8
--- parted-1.6.12.orig/doc/parted.8	2004-08-31 11:43:35.000000000 +0200
+++ parted-1.6.12/doc/parted.8	2004-08-31 11:45:35.000000000 +0200
@@ -23,8 +23,8 @@
 .SH DESCRIPTION
 This manual page documents briefly the
 .BR parted
-command.  Complete documentation is distributed with the package in
-GNU Info format; see below.
+command.  Complete documentation is distributed with the parted-doc Debian
+package in GNU Info format; see below.
 .PP
 .B parted
 is a disk partitioning and partition resizing program.  It allows you to
@@ -37,10 +37,10 @@
 displays a help message.
 .TP
 .B -i, --interactive
-where necessary, prompts for user intervention.
+prompt the user where necessary.
 .TP
 .B -s, --script
-never prompts for user intervention.
+never prompt the user.
 .TP
 .B -v, --version
 displays the version.
diff -urNda parted-1.6.12.orig/docpt_BR-parted.8 parted-1.6.12/doc/pt_BR-parted.8
--- parted-1.6.12.orig/docpt_BR-parted.8	2004-08-31 11:43:35.000000000 +0200
+++ parted-1.6.12/doc/pt_BR-parted.8	2004-08-31 11:44:43.000000000 +0200
@@ -59,10 +59,10 @@
 .\" displays a help message.
 .\" .TP
 .\" .B -i, --interactive
-.\" where necessary, prompts for user intervention.
+.\" prompt the user where necessary
 .\" .TP
 .\" .B -s, --script
-.\" never prompts for user intervention.
+.\" never prompt the user
 .\" .TP
 .\" .B -v, --version
 .\" displays the version.
